platform/solr/solr-dependencies/pom.xml

```diff
@@ -21,199 +21,627 @@
-                            <include>org.apache.lucene:lucene-suggest</include>
-                            <include>org.noggit:noggit</include>
-                            <include>org.restlet.jee:org.restlet</include>
-                            <include>org.restlet.jee:org.restlet.ext.servlet</include>
-                            <include>com.facebook.presto:presto-parser</include>
-                            <include>com.google.protobuf:protobuf-java</include>
-                            <include>org.rrd4j:rrd4j</include>
-                            <include>io.sgr:s2-geometry-library-java</include>
-                            <include>org.apache.solr:solr-core</include>
-                            <include>org.apache.solr:solr-solrj</include>
-                            <include>org.locationtech.spatial4j:spatial4j</include>
-                            <include>com.tdunning:t-digest</include>
-                            <include>com.fasterxml.woodstox:woodstox-core</include>
-                            <include>org.apache.zookeeper:zookeeper</include>
-                            <include>io.dropwizard.metrics:metrics-jetty9</include>
-                            <include>io.dropwizard.metrics:metrics-graphite</include>
-                            <include>io.dropwizard.metrics:metrics-ganglia</include>
-                            <include>io.dropwizard.metrics:metrics-jvm</include>
-                            <include>io.dropwizard.metrics:metrics-core</include>
-                            <include>info.ganglia.gmetric4j:gmetric4j</include>
-                            <include>org.slf4j:slf4j-log4j12</include>
-                            <include>org.slf4j:jul-to-slf4j</include>
-                            <include>org.slf4j:jcl-over-slf4j</include>
-                            <include>log4j:log4j</include>
-                        </includes>
-                    </artifactSet>
-                    <filters>
-                        <filter>
-                            <artifact>*:*</artifact>
-                            <excludes>
-                                <exclude>META-INF/*.SF</exclude>
-                                <exclude>META-INF/*.DSA</exclude>
-                                <exclude>META-INF/*.RSA</exclude>
-                                <exclude>module-info.class</exclude>
-                            </excludes>
-                        </filter>
-                    </filters>
-                    <transformers>
-                        <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
-                    </transformers>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Require-Capability>osgi.ee;filter:="(&amp;(osgi.ee=JavaSE)(version=1.8))"</Require-Capability>
+                        <Export-Package>
```

Comment:
❗️ I think we should export the proper library versions.  Otherwise these will be exported as our project version instead.

Reply:
Great catch, they manifest showed the some of the packages with ddf versions. I've added the correct export versions.